### DISCUSSION TITLE
Invalid Manifest error

### DISCUSSION THREAD

**Discussion by DenverDaveH** [Able-Bear-Software/tab-nodes-tree#28] [General]
Hello,

Am trying to test out this as my co. has moved to Chrome. Bummer.

After reading the reviews, I am clicking on the Add To Chrome button. I get a popup in the lower left hand corner:

Download error: Invalid Manifest.

Thoughts?

**Comment by zeroarst** [comment]
I haven't seen other users report this issue yet, and I just performed a fresh install using a different user profile, which worked successfully. To assist you better, could you please provide a screenshot of the error message along with your Chrome version and operating system details? This information will help me troubleshoot the problem more effectively.

Thank you!

**Comment by DenverDaveH** [comment]
Well call me a suck egg mule!

I went to get the Chrome version and it forced an update. I am now on:
Version 128.0.6613.138 (Official Build) (64-bit)

And it installed!!

Now let's see if my corporate overlords will allow me to install it as I am being forced of FF and onto Chrome. And I love the FF tree style tabs extension!!

We can close this issue as the update seemed to resolve the extension install problem.

Thanks!
